ChatApp/Custom+UI/TextFieldContainerView.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  TextFIeldContainer.swift
+//  ChatApp
+//
+//  Created by bakhva  on 27.04.23.
+//
+
+import UIKit
+
+
+final class TextFieldContainerView: UIView {
+
+    let textField = ScrollableTextField()
+    var sendButton = UIButton(type: .system)
```

Comment:
I guess it will be better to use closures to configure and initialize this button here.

also, make it private